tests/model/test_model.py

```diff
@@ -612,6 +612,21 @@ def test_success_get_messages(self, mocker, messages_successful_response,
             assert model.index['pointer'][repr(model.narrow)] == anchor
         assert model._have_last_message[repr(model.narrow)] is True
 
+    @pytest.mark.parametrize('messages, expected_messages_response', [(
+        [{'subject_links': ['www.foo1.com', 'www.bar1.com']}],
+        [{'topic_links':   ['www.foo1.com', 'www.bar1.com']}]), (
+        [{'topic_links':   ['www.foo2.com', 'www.bar2.com']}],
+        [{'topic_links':   ['www.foo2.com', 'www.bar2.com']}]),
+    ], ids=[
+        'param_old_server_release',
+        'param_new_server_release',
```

Comment:
It may be worth adding some empty cases too? The code should already handle it, and the test will then specify that it should.

Reply:
By empty do you mean `list()` or `None`? We don't have a `None` value response for `topic_links` from API call anyways. That said I'll add some empty list tests. Thanks for pointing out :)